src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -263,13 +263,9 @@ private Document GetDocument()
 
         private CompletionHelper GetCompletionHelper()
         {
+            // Crash if we don't find a document, we're already in a bad state.
```

Comment:
Should you actually just crash here instead of seemingly relying on `CompletionHelper.GetHelper` to do it for you? What if somebody updates `CompletionHelper.GetHelper` without realizing this code wants it to crash on null?